CHANGELOG.rst

```diff
@@ -0,0 +1,97 @@
+0.13.2
+------
+Unreleased
+
+- Added support for recursively terminating entire process tree with
+  :meth:`XProcessInfo._terminate_children()`
```

Comment:
Not sure we should be documenting the internal function, we probably should make it clear that the default `XProcessInfo.terminate` has changed. How about:

```rst
- Now ``XProcessInfo.terminate`` will by default also terminate the entire process tree. This is safer as there's no risk of leaving lingering processes behind. If for some reason you need the previous behavior of only terminating the root process, ```kill_proc_tree=True`` to ``XProcessInfo.terminate``.
```

Also this is a new feature, so this should be `0.14` (technically it is a breaking change, but we are still in `0.X` so just bumping the minor number should be fine).